diff --git a/config.def.h b/config.def.h
index 51a4a94..f147c44 100644
--- a/config.def.h
+++ b/config.def.h
@@ -145,8 +145,9 @@ static const Key keys[] = {
 	{ MODKEY,                       XK_r,      setlayout,      {.v = &layouts[3]} },
 	{ MODKEY,                       XK_g,      setlayout,      {.v = &layouts[4]} },
 	{ MODKEY|ShiftMask,             XK_g,      rldecktogglealt, {0} },
-	{ MODKEY,                       XK_o,      rldeckrotate,   {.i = +1 } },
-	{ MODKEY|ShiftMask,             XK_o,      rldeckrotate,   {.i = -1 } },
+	{ MODKEY|ShiftMask,             XK_r,      rldecktogglealt, {0} },
+	{ MODKEY,                       XK_v,      rldeckrotate,   {.i = +1 } },
+	{ MODKEY|ShiftMask,             XK_v,      rldeckrotate,   {.i = -1 } },
 	{ MODKEY,                       XK_space,  setlayout,      {0} },
 	{ MODKEY|ShiftMask,             XK_space,  togglefloating, {0} },
 	{ MODKEY,                       XK_0,      view,           {.ui = ~0 } },
